fast_forward/universe_handler.py

```diff
@@ -40,6 +41,14 @@ def __init__(self, mol_names, *args, **kwargs):
         super().__init__(*args, **kwargs)
         # select which molecules to treat
         self.mol_names = mol_names
+
+        # We copy residue info as moleculetype info, if it's missing.
+        if not hasattr(self.atoms, "moltypes"):
+            moltypes = Moltypes(self.residues.resnames)
+            molnums = Molnums(range(len(self.residues)))
+            self.add_TopologyAttr(moltypes)
+            self.add_TopologyAttr(molnums)
```

Comment:
@csbrasnett @mnmelo  this would probably make the treatment in `itp_to_ag` redundant, right? So we could remove line https://github.com/fgrunewald/fast_forward/blob/81615115b367eba72517ef8307c39f74927bf449/fast_forward/itp_to_ag.py#L52-L62

Reply:
I think the current handling is already much better than before. We'll keep it for later to clean this up more